content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -121,9 +121,7 @@ If the database provider is MySQL, the values are `1` or `0`. However, if the da
 
 > **Note**: The Prisma Client query engine standardizes the return type for all databases. **Using the raw queries does not**.
 
-## <inlinecode>$queryRawUnsafe</inlinecode>
-
-<a id="query-raw-unsafe"></a>
+## <inlinecode>$queryRawUnsafe</inlinecode> {#query-raw-unsafe}
```

Comment:
Whoah!? What kinda of magic is this? How does this work?

Reply:
Oh, it unfortunately does not in the ToC 😢 
![image](https://user-images.githubusercontent.com/183673/133248923-2adcded7-571d-4b26-bf48-09ba60857b7e.png)

Reply:
Boooo

Reply:
😆

Reply:
Can we not just get rid of the anchors with `-` and use `#queryrawunsafe in the links? Is there a downside to that?

Reply:
I just used the full path, even though its in the same file and it now works to navigate to the header in the same file. Whats odd here is that we link to headers with inlinecode in other places in the docs. Anyway, this works so im leaving it at that 😅